manager/manager.go

```diff
@@ -111,8 +111,7 @@ type Config struct {
 // This is the high-level object holding and initializing all the manager
 // subsystems.
 type Manager struct {
-	config    *Config
-	listeners []net.Listener
+	config Config
```

Comment:
> Why the change from pointer to value?

We now modify `m.config` in `BindRemote` and `BindControl`, and we want these modifications to be private to the manager, rather than potentially overwriting fields in a structure that was passed in by something else.